person.rb

```diff
@@ -7,6 +7,7 @@ def initialize(age, name: 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Getters and Setters for @rentals should be present in the `Person` class. Kindly add that to the already set methods. It should look something like this.

`attr_accessor :name, :age, @rentals`